plugin/storage/es/options.go

```diff
@@ -147,8 +146,8 @@ func addFlags(flagSet *flag.FlagSet, nsConfig *namespaceConfig) {
 		"Timeout used for queries. A Timeout of zero means no timeout")
 	flagSet.Duration(
 		nsConfig.namespace+suffixMaxSpanAge,
-		nsConfig.MaxSpanAge,
-		"The maximum lookback for spans in Elasticsearch")
+		time.Hour*72,
+		"(deprecated) The maximum lookback for spans in Elasticsearch. Now all indices are searched.")
```

Comment:
This seems confusing. Can we add the message `Now all indices are searched` in the parenthesis?